stable/voyager/templates/cluster-role.yaml

```diff
@@ -0,0 +1,69 @@
+{{ if .Values.rbac.create }}
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    app: "{{ template "name" . }}"
+    heritage: "{{ .Release.Service }}"
+    release: "{{ .Release.Name }}"
+rules:
+- apiGroups:
+  - extensions
+  resources:
+  - thirdpartyresources
```

Comment:
Maybe the custom resources should just be part of the helm chart? That way Voyager wouldn't need the ability to create them.

Reply:
Voyager operator automatically registers TPR if that is not present. I think discussion for changes like this are out of scope for this type of prs.

Reply:
:+1: works for me.